### ISSUE TITLE
NullPointerException with multipart/form-data requests on "io.gatling.gradle" 3.4.0 and 3.4.1

### ISSUE THREAD

**Issue by savvagen** [gatling/gatling#3995]
## Summary
Hello! I got NullPointer requests while switching to gatling gradle plugins:
from: `id "com.github.lkishalmi.gatling" version "3.3.4"`
to: `id "io.gatling.gradle" version "3.4.0"` or `id "io.gatling.gradle" version "3.4.1"`

Exception body:
```
11:36:39.928 [INFO ] i.g.h.e.GatlingHttpListener$ - Request 'Create Attachment' failed for user 1
java.lang.NullPointerException: null
	at io.gatling.http.client.body.multipart.Part.toString(Part.java:105)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at io.gatling.http.client.body.multipart.MultipartFormDataRequestBody.lambda$toString$0(MultipartFormDataRequestBody.java:99)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at io.gatling.http.client.body.multipart.MultipartFormDataRequestBody.toString(MultipartFormDataRequestBody.java:99)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at io.gatling.http.util.package$HttpStringBuilder$.$anonfun$appendRequest$2(package.scala:61)
	at scala.Option.foreach(Option.scala:407)
	at io.gatling.http.util.package$HttpStringBuilder$.appendRequest$extension(package.scala:60)
	at io.gatling.http.engine.response.StatsProcessor.dump$1(StatsProcessor.scala:74)
	at io.gatling.http.engine.response.StatsProcessor.logTx(StatsProcessor.scala:89)
	at io.gatling.http.engine.response.StatsProcessor.reportStats(StatsProcessor.scala:39)
	at io.gatling.http.engine.response.DefaultResponseProcessor.proceed(ResponseProcessor.scala:87)
	at io.gatling.http.engine.response.DefaultResponseProcessor.handleResponse(ResponseProcessor.scala:79)
	at io.gatling.http.engine.response.DefaultResponseProcessor.onComplete(ResponseProcessor.scala:57)
	at io.gatling.http.engine.GatlingHttpListener.onHttpResponseBodyChunk(GatlingHttpListener.scala:155)
	at io.gatling.http.client.impl.HttpAppHandler.channelRead(HttpAppHandler.java:193)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)

11:46:20.632 [INFO ] i.g.h.e.r.DefaultStatsProcessor - Request 'Create Attachment' failed for user 1: j.l.NullPointerException
11:46:20.633 [DEBUG] i.g.c.a.Exit - End user #1
11:46:20.639 [DEBUG] i.g.c.c.i.Injector - End user #My Scenario
11:46:20.639 [ERROR] i.g.h.c.i.HttpAppHandler - Exception while handling HTTP/1.1 crash, please report to Gatling maintainers
java.lang.NullPointerException: null
	at io.gatling.http.client.body.multipart.Part.toString(Part.java:105)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at io.gatling.http.client.body.multipart.MultipartFormDataRequestBody.lambda$toString$0(MultipartFormDataRequestBody.java:99)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at io.gatling.http.client.body.multipart.MultipartFormDataRequestBody.toString(MultipartFormDataRequestBody.java:99)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at io.gatling.http.client.Request.toString(Request.java:178)
	at java.lang.String.valueOf(String.java:2994)
	at scala.collection.mutable.StringBuilder.append(StringBuilder.scala:203)
	at scala.collection.TraversableOnce$appender$1$.apply(TraversableOnce.scala:395)
	at scala.collection.TraversableOnce$appender$1$.apply(TraversableOnce.scala:391)
	at scala.collection.Iterator.foreach(Iterator.scala:943)
	at scala.collection.Iterator.foreach$(Iterator.scala:943)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
	at scala.collection.TraversableOnce.addString(TraversableOnce.scala:404)
	at scala.collection.TraversableOnce.addString$(TraversableOnce.scala:389)
	at scala.collection.AbstractIterator.addString(Iterator.scala:1431)
	at scala.collection.TraversableOnce.mkString(TraversableOnce.scala:359)
	at scala.collection.TraversableOnce.mkString$(TraversableOnce.scala:358)
	at scala.collection.AbstractIterator.mkString(Iterator.scala:1431)
	at scala.runtime.ScalaRunTime$._toString(ScalaRunTime.scala:170)
	at io.gatling.http.response.HttpFailure.toString(Response.scala:34)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at io.gatling.http.engine.response.DefaultResponseProcessor.handleFailure(ResponseProcessor.scala:68)
	at io.gatling.http.engine.response.DefaultResponseProcessor.onComplete(ResponseProcessor.scala:58)
	at io.gatling.http.engine.GatlingHttpListener.onThrowable(GatlingHttpListener.scala:212)
	at io.gatling.http.client.impl.HttpAppHandler.crash(HttpAppHandler.java:81)
	at io.gatling.http.client.impl.HttpAppHandler.channelRead(HttpAppHandler.java:196)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

## How to reporduce
Reproduced on simple simmulation:
```
class ExampleSimulation extends Simulation {

  var httpConf = http.baseUrl("<baseUrl>")
    .acceptCharsetHeader("utf-8")
    .acceptHeader("application/json")
    .disableWarmUp
    .disableCaching

  val itemId ="<id>"
  val projectId ="<id>"
  val authToken = "<token>"

  val failScenario = scenario("My Scenario").exec(myRequest)

  def myRequest= {
    http("Create Attachment").post("/api/attachment")
      .header("Authorization", s"Token ${authToken}")
      .header("Content-Type", "multipart/form-data")
      .formParam("type", "test")
      .formParam("object", f"${itemId}")
      .formUpload("file", s"${dataFolder}/attachment.png")
      .formParam("project", projectId)
      .check(status.is(201))
  }

  setUp(failScenario.inject(atOnceUsers(1))).protocols(httpConf)

}

```
### How to fix

When I switched back to Gradle plugin: 
```
plugins {
  id "com.github.lkishalmi.gatling" version "3.3.4"
}

gatling {
  toolVersion = "3.3.0"
}
```
It started working, and request is OK!

**Comment by slandelle** [comment]
Thanks for reporting!

**Comment by savvagen** [comment]
@slandelle Thanks!) Issue is fixed in Gatling - 3.4.2
